CDP4ShellDialogs/ViewModels/PluginManagerViewModel.cs

```diff
@@ -0,0 +1,174 @@
+﻿// --------------------------------------------------------------------------------------------------------------------
+// <copyright file="PluginManagerViewModel.cs" company="RHEA System S.A.">
+//   Copyright (c) 2015-2018 RHEA System S.A.
```

Comment:
Small thing: When changes are made, also change the header. In this case 2015-2020.